templates/shared_services/firewall/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-shared-service-firewall
-version: 1.3.3
+version: 1.4.0
 description: "An Azure TRE Firewall shared service"
```

Comment:
I understand we can't change the name, but we can consider changing the description, template schema and docs to reflect the fact this is now just about managing the workspace level firewall policy rules.

Reply:
Yup, primarily wanted to see if thought this works as a change in principal. If agreed, will go through and fix up.

Reply:
I've changed in the porter file, can't see any references in the docs.